import{j as t}from"./jsx-runtime-DNP5ZHPF.js";import{m as h}from"./mockTableEntity-DOIjzTTt.js";import{g as m,r as u}from"./tableQueryHandlers-CP-eZ2G2.js";import{M as i}from"./useFiles-DxEFRl-N.js";import{t as p,k as f,i as g,j as y,m as b,l as w,f as v}from"./SynapseConstants-x7wBmLwm.js";import"./index-C9LqO53Z.js";import"./VerificationSubmission-B0kBNeMy.js";import"./StringUtils-CzgJJW-6.js";import"./OrientationBanner-lGnlzz1n.js";import{l as N}from"./index-Bjvko-XC.js";import{Q as F,a as T}from"./QueryWrapper-DutBOCKd.js";import{S}from"./SynapseTable-BeUht_9T.js";import"./mock_user_profile-CijvmO8r.js";import"./mockProject-CtkMX8E8.js";import"./mockTeam-BZngCltQ.js";import"./uniqueId-CTU4ZW8o.js";import"./toString-Drlpq-aN.js";import"./isObjectLike-BAT6WrGj.js";import"./iframe-DzZkUbt0.js";import"./isArray-Dxzbedgu.js";import"./isSymbol-CR6CJbsv.js";import"./times-ChNYGLHd.js";import"./_baseTimes-36S_kd0L.js";import"./identity-DKeuBCMA.js";import"./toInteger-NHA3mITb.js";import"./asyncJobHandlers-DKIwM4P_.js";import"./BasicMockedCrudService-DXeqISkl.js";import"./cloneDeep-DGtsa-PL.js";import"./_Uint8Array-CYf3IgYh.js";import"./_initCloneObject-BZtj6YT_.js";import"./_getTag-DKMcyan0.js";import"./isEqual-CyOrkakg.js";import"./_baseIsEqual-DUOouxQQ.js";import"./omit-CWFUit5V.js";import"./_baseGet-Gev72UCi.js";import"./isNil-C_6EL5i8.js";import"./last-DXTTfs9a.js";import"./_baseSlice-F8doVSIJ.js";import"./merge-CAVgou7m.js";import"./_baseFlatten-Zu_cgFPk.js";import"./fetchWithExponentialTimeout-Cpvpsfq8.js";import"./TransitionGroupContext-CdwE7o-d.js";import"./FullWidthAlert-XMCCRjsq.js";import"./spreadSx-CwcO6WA9.js";import"./Alert-DBftbUxX.js";import"./createTheme-BG9REND8.js";import"./resolveComponentProps-DJsKa-oF.js";import"./isHostComponent-DVu5iVWx.js";import"./useForkRef-D7H_VIUT.js";import"./createSvgIcon-BtT08fNl.js";import"./DefaultPropsProvider-CBoC3p-_.js";import"./Close-BrzxmRIz.js";import"./IconButton-HsPug_ob.js";import"./ButtonBase-C0iYjrZq.js";import"./useTimeout-9s3akjMN.js";import"./useIsFocusVisible-C4iYqwwm.js";import"./useEventCallback-CFsLaU7U.js";import"./Paper-BZJIjrog.js";import"./Stack-C3Yli3Do.js";import"./getThemeProps-CmxBhP06.js";import"./useTheme-Dqg7276U.js";import"./Box-DhP9mjg7.js";import"./AlertTitle-CFko3bX9.js";import"./Typography-BdVBJyiw.js";import"./useTheme-DaIKVKIj.js";import"./Grow-BEsXXeLg.js";import"./index-B7SGfn2n.js";import"./utils-Bn3sr0WI.js";import"./ClickAwayListener-CiZf8GXf.js";import"./ownerDocument-DW-IO8s5.js";import"./Tooltip--gcNUJrW.js";import"./index-DfJVQJTe.js";import"./useControlled-C6FPOIeJ.js";import"./useId-Bp9otoFB.js";import"./Popper-wFK3a8ms.js";import"./Button-C1D6jijN.js";import"./useQuery-Drm9MnN6.js";import"./utils-Dge9E0v5.js";import"./Link-BP8bljuP.js";import"./Collapse-DAEF_CSM.js";import"./tinycolor-Begke6kS.js";import"./Fade-BS8huG_s.js";import"./Skeleton-X6vAixA1.js";import"./inputBaseClasses-Cvq7yWxw.js";import"./calculateFriendlyFileSize-FpJgzzI_.js";import"./CheckCircleTwoTone-DooSA5RY.js";import"./InfoTwoTone-BUvN6rod.js";import"./mutation-DZbFfGts.js";import"./dayjs.min-DYHBVdyC.js";import"./chunk-AYJ5UCUI-BAVTTsGw.js";import"./index-DNCRZXKi.js";import"./react-LeG2f8XJ.js";import"./util-D0jHV5TN.js";import"./unCamelCase-ZiU_emRy.js";import"./use-deep-compare-effect.esm-CGTFEkO2.js";import"./QueryContext-DfvEq5GV.js";import"./NoSearchResults-CW8xfM1u.js";import"./NoData-DW9zp9Wg.js";import"./SqlFunctions-DnxC6kpb.js";import"./index-CQWDxYyK.js";import"./index-D8bDFOii.js";import"./noop-DX6rZLP_.js";import"./ConfirmationDialog-BvQyTyJF.js";import"./DialogBase-CiljmXUt.js";import"./Close-CxYKehz7.js";import"./createSvgIcon-Bayh6HlG.js";import"./createChainedFunction-BO_9K8Jh.js";import"./debounce-Be36O1Ab.js";import"./isMuiElement-BMn3X3Gz.js";import"./ownerWindow-HkKU3E4x.js";import"./HelpPopover-CuENmKTA.js";import"./MarkdownPopover-BYIzAOpc.js";import"./LightTooltip-CmYRAIyM.js";import"./MarkdownSynapse-BeO0RyoA.js";import"./SkeletonButton-BK7YgQCa.js";import"./SkeletonInlineBlock-D4jApRcZ.js";import"./SkeletonTable-Bik19zmH.js";import"./SkeletonParagraph-DK8LxRG0.js";import"./HelpOutlineTwoTone-Bz7TfZfC.js";import"./DialogTitle-CC3UEf9Q.js";import"./DialogContent-CXXeVq0f.js";import"./Dialog-Cab871qq.js";import"./Modal-bbWkmLnK.js";import"./getScrollbarSize-Bqq2hMjQ.js";import"./Backdrop-UeKywpjz.js";import"./TableRowSelectionState-DJuPbdhA.js";import"./useEntity-DqG3EDzd.js";import"./includes-DaeJ4RJy.js";import"./_baseIndexOf-rBnIfvEs.js";import"./isString-JI22sBIr.js";import"./_baseIteratee-DwKy7MgG.js";import"./hasIn-D0DPTVmN.js";import"./pickBy-JbFxjxTN.js";import"./pick-Dj6JmVag.js";import"./_baseSet-Cn-LZY_U.js";import"./useQueries-D0WgMyT9.js";import"./useMutation-DTkKfOYq.js";import"./useInfiniteQuery-Bv25vp6V.js";import"./InfiniteQueryUtils-CKlRW-xB.js";import"./QueryFilterUtils-Djjqq8EN.js";import"./isNumber-DGe9QvaL.js";import"./useEntityBundle-DvRMGHeP.js";import"./SynapseTableUtils-DSUAM55b.js";import"./StyledTanStackTable-fdH_iri2.js";import"./StyledTableContainer-Cz1PcxVC.js";import"./ExpandCollapseButton-DROmRiMG.js";import"./IconSvg-DVy36fcc.js";import"./LayersTwoTone-CwImyUqN.js";import"./ContentCopyTwoTone-D_B6wDzg.js";import"./ErrorOutlined-CfYzmiSd.js";import"./GetAppTwoTone-CFy4yj9d.js";import"./InfoOutlined-Q8-fM4At.js";import"./DeleteTwoTone-DNYgeBSo.js";import"./ModalDownload-gqRGhleY.js";import"./index-DGUIYPz9.js";import"./enums-CQItE0R5.js";import"./_arrayReduce-CrxnWFSq.js";import"./_baseEach-DlDHe4Hi.js";import"./uniq-COuL0TC_.js";import"./_arrayIncludesWith-DtPV-o4u.js";import"./forEach-C4yctKF1.js";import"./_baseDifference-BI8IXiUv.js";import"./index.browser-CtiQQ-iP.js";import"./Grid-SBbQYNEI.js";import"./TextField-NRUXNsvd.js";import"./useFormControl-BVCngBGa.js";import"./formControlState-Dq1zat_P.js";import"./InputLabel-CYXBXRDP.js";import"./FormControl-Cffw6pI4.js";import"./FormLabel-Bt6Vk9aB.js";import"./Input-CDON1sZ_.js";import"./Menu-BlR8J5db.js";import"./Popover-BgSor5FC.js";import"./MenuList-ERIzIIsf.js";import"./List-dJnqPd28.js";import"./ListItem-aPQuXonM.js";import"./listItemButtonClasses-B8TitDYl.js";import"./ListItemIcon-DKFzcGct.js";import"./MenuItem-aQW3zzeO.js";import"./dividerClasses-BVg1RLGC.js";import"./ListItemText-D6qFxAQB.js";import"./ContentCopy-BVYCG1gH.js";import"./Divider-agOjyXuA.js";import"./FormControlLabel-Bz54JIbQ.js";import"./Checkbox-BRH9nGgz.js";import"./SwitchBase-1hFInEzP.js";import"./FormGroup-CrdKqTsB.js";import"./RadioGroup-JxarJqxy.js";import"./Radio-xJemyZ9P.js";import"./Slider-DTDxiYRc.js";import"./SynapseFormRadioWidget-D0uVEucq.js";import"./useGetEntityHeaders-DxbsW9a0.js";import"./useDownloadList-Dc0wldCd.js";import"./ToastMessage-C3sj4-_Q.js";import"./CSSTransition-3Ix0o81q.js";import"./FileEntityDirectDownload-B__qy_Kl.js";import"./DirectDownload-Cvh_6yd2.js";import"./SynapseTableConstants-2qH3fDlQ.js";import"./HasAccessV2-Bn93lbiv.js";import"./chunk-C4WEiA25.js";import"./AccessRequirementList-D8mM9Z5i.js";import"./ACTAccessRequirement-WDpE4gfi.js";import"./SelfSignAccessRequirement-C_01XAIc.js";import"./useAccessRequirements-BiOU9BIr.js";import"./useMediaQuery-CKYe7vz_.js";import"./_baseOrderBy-rh7Yuney.js";import"./_baseMap-BS6rNZ9j.js";import"./groupBy-T42DzHa9.js";import"./_createAggregator-DZnaBYtv.js";import"./StandaloneLoginForm-D0dW_6Dd.js";import"./useTwoFactorEnrollment-Dgona7rf.js";import"./LoginMethodButton-DaSbMkWl.js";import"./LastLoginInfo-yfgpNOkv.js";import"./DateFormatter-CZB-plYw.js";import"./utc-B0DH-1ej.js";import"./localizedFormat-HD4camVj.js";import"./TextField-B4pLpVaz.js";import"./StyledFormControl-DeJIcWuC.js";import"./EntityLink-C74S0T4r.js";import"./EntityIcon-Bha78Vx3.js";import"./ErrorChip-MnzaN1GQ.js";import"./Chip-oRUPqFia.js";import"./Avatar-g_KuDRey.js";import"./UserOrTeamBadge-CU9vYDjl.js";import"./UserBadge-UtVQL_Wk.js";import"./Card-C2SfOBfp.js";import"./TeamBadge-DozxxPe1.js";import"./UnmanagedACTAccessRequirementItem-CXRK7wVp.js";import"./RequirementItem-016-fk3c.js";import"./ConditionalWrapper-CdyME2LR.js";import"./LockTwoTone-DFaz6q8w.js";import"./SelfSignAccessRequirementItem-CdiXSv6r.js";import"./DataAccessRequestAccessorsFilesForm-B1FBczgG.js";import"./useDataAccessSubmission-D1CdcC7k.js";import"./UserSearchBoxV2-wd0qOHzK.js";import"./useDebouncedEffect-DArt0bOz.js";import"./without-D_O_I6D8.js";import"./Select-aab027f3.esm-Bq1L_ySV.js";import"./defineProperty-Cp4flQ_6.js";import"./toPropertyKey-C9c9JG92.js";import"./UploadDocumentField-BqY1AA5m.js";import"./FileUpload-aXOmOMPK.js";import"./ManagedACTAccessRequirementFormWikiWrapper-RdwjFnyP.js";import"./ResearchProjectForm-B-_a97QB.js";import"./TextFieldWithWordLimit-lswCsnsX.js";import"./AuthenticatedRequirement-B4lxu4Do.js";import"./CertificationRequirement-9X98b1Am.js";import"./TwoFactorAuthEnabledRequirement-tv94VOl4.js";import"./TwoFactorEnrollmentForm-Bj_1ExPZ.js";import"./LeftRightPanel-CNJvyxOO.js";import"./LoadingScreen-60maNtYR.js";import"./LinearProgress-WoOq-nai.js";import"./ValidationRequirement-BI_KTMPD.js";import"./ColumnHeader-BkDSJpOf.js";import"./EnumFacetFilterUI-CDf4Oh54.js";import"./FacetFilterHeader-CnsTifTH.js";import"./HelpTwoTone-D9quyIqy.js";import"./EnumFacetFilter-C0HlhAfE.js";import"./EnumFacetFilterSkeleton-BUUFXtTZ.js";import"./InteractiveCopyIdsIcon-CFC02ZEH.js";import"./SynapseTableCell-CZOL8iv7.js";import"./SynapseCardLabel-BMvqNTrO.js";import"./CardUtils-CHxXfKty.js";import"./ImageFileHandle-5PN7fElA.js";import"./index-D0TM4b_V.js";import"./Box-B8b8fC6H.js";import"./GenericCard-4lHfKfPE.js";import"./HeaderCard-HjbLLTHJ.js";import"./index-CUpalX01.js";import"./Icon-BnSlBR3D.js";import"./mouse-DF4kmUNB.js";import"./SortDown-DJ15V9bx.js";import"./ShowMore-DGu-Ldxl.js";import"./HeaderCardV2-B3TZ4Lwz.js";const n={concreteType:"org.sagebionetworks.repo.model.table.QueryResultBundle",queryCount:270,selectColumns:[{name:"studyName",columnType:"LARGETEXT",id:"82659"},{name:"studyId",columnType:"ENTITYID",id:"82658"},{name:"summary",columnType:"LARGETEXT",id:"82091"},{name:"accessRequirements",columnType:"LARGETEXT",id:"111501"},{name:"acknowledgementStatements",columnType:"LARGETEXT",id:"111502"},{name:"id",columnType:"STRING",id:"82871"}],maxRowsPerPage:238,columnModels:[{name:"studyName",columnType:"LARGETEXT",id:"82659"},{name:"studyId",columnType:"ENTITYID",id:"82658"},{name:"summary",columnType:"LARGETEXT",id:"82091"},{name:"accessRequirements",columnType:"LARGETEXT",id:"111501"},{name:"acknowledgementStatements",columnType:"LARGETEXT",id:"111502"},{name:"id",columnType:"STRING",id:"82871",maximumSize:50}],facets:[],sumFileSizes:{sumFileSizesBytes:0,greaterThan:!1},queryResult:{concreteType:"org.sagebionetworks.repo.model.table.QueryResult",queryResults:{concreteType:"org.sagebionetworks.repo.model.table.RowSet",tableId:"syn16787123",etag:"58a859ba-e4b2-41bf-8354-4b8a6da0a4fd",headers:[{name:"studyName",columnType:"LARGETEXT",id:"82659"},{name:"studyId",columnType:"ENTITYID",id:"82658"},{name:"summary",columnType:"LARGETEXT",id:"82091"},{name:"accessRequirements",columnType:"LARGETEXT",id:"111501"},{name:"acknowledgementStatements",columnType:"LARGETEXT",id:"111502"},{name:"id",columnType:"STRING",id:"82871"}],rows:[{rowId:499,versionNumber:897,values:["A 3D Cutaneous Neurofibroma Model for Automated High-Throughput Drug Screenings","syn11374354","Given the absence of therapeutic options and the heterogeneous nature of cutaneous neurofibromas (cNF), a physiological system that supports the growth of cNF tumor cells is required for drug screening efforts. We aim to adapt our optimized high-throughput 3D ring approach to develop a novel preclinical cNF model.  My laboratory has developed a robust high-throughput system to establish tumor organoids using a specific geometry to minimize sample handling and cell seeding density. By seeding cells in Matrigel in a mini-ring format we are able to routinely generate organoids from cell lines and primary samples in a format suitable for automation and high-throughput drug screenings. The approach does not necessitate of a priori cell expansion and allows to pinpoint drug sensitivities in within 5-6 days, hence avoiding most of the issues that arise from sample senescence and/or accumulation of genetic changes that can occur with prolonged time in culture. Our method is particularly suited to sustain growth of clinically relevant heterogeneous primary samples, both malignant as well as benign such as Schwann cells or fibroblasts. A single clinical sample can provide sufficient material to screen hundreds of drugs and identify individual drug susceptibilities. Here we aim to extend our proven method to cNF specimens to establish a reliable pre-clinical model. We will use primary samples obtained from NF1 patients and establish cNF organoids to elucidate disease biology and enable therapeutic testing. We anticipate collecting n=10 samples from adult NF1 patients through established collaborations with the UCLA NF clinic. We will characterize the tissues by histopathology and next generation sequencing (NGS) and determine whether the cNF 3D models recapitulate all the major features of the tumor of origin. While beyond the scope of the current project, the cNF models developed and validated here may be used in the future to perform automated high throughput 3D drug screenings to identify possible new therapeutic options for NF1 patients.","Accessing the data from this study requires a Synapse account.","Please acknowledge the NF Data Portal like so: “The results published here are in whole or in part based on data obtained from the NF Data Portal (http://www.nf.synapse.org, RRID:SCR_021683) and made available through the NF Open Science Initiative.”","syn11374354"]},{rowId:500,versionNumber:897,values:["Synodos NF2","syn2343195","Synodos for NF2 is a first-of-its-kind neurofibromatosis research collaboration dedicated to defeating the rare genetic disorder neurofibromatosis type 2 (NF2). This unique consortium brings together a multidisciplinary team of scientists from twelve world-class labs at academic and medical centers of excellence, who have pledged to work closely together - sharing information, datasets, results and more - at every step in research development, with the goal of speeding up the drug discovery process. The end goal of this effort is to find new approaches to the diagnosis and treatment of two primary NF2-related tumors: schwannoma and meningioma. The expectation is to take the work from discovery to pre-clinical development, resulting in the development of an Investigational New Drug (IND) for NF2. Presently, two of three data releases have been conducted, with high-throughput drug screening data and more gene expression/kinomics data to be released in the future.","Accessing the data from this study requires a Synapse account.",'If you use data from this study in a publication or talk, please cite the following publication: "The Synodos for NF2 Consortium, et al. (2018) Traditional and systems biology based drug discovery for the rare tumor syndrome neurofibromatosis type 2. PLoS ONE 13(6): e0197350. https://doi.org/10.1371/journal.pone.0197350." In addition, please acknowledge the NF Data Portal using the following statement: "The results published here are in whole or in part based on data obtained from the NF Data Portal (http://www.nf.synapse.org, RRID:SCR_021683) and made available through the NF Open Science Initiative."',"syn2343195"]},{rowId:501,versionNumber:897,values:["The Johns Hopkins NF1 biospecimen repository","syn4939902","Surgery is currently the only treatment option for patients with NF1 with symptomatic plexiform neurofibromas (pNF). Progress developing nonsurgical therapy for pNF has been limited by a number of factors including 1) the lack of cell culture based models of pNF, 2) a limited number of animal models of pNF, and 3) limited access of investigators to primary pNF tissue from NF1 patients. Although progress is being made in the development and utilization of animal models and cell culture models, the limited availability of primary patient tissue remains a gap. The goal of this project is to establish a biospecimen and xenograft bank from NF1 patients with pNF and malignant peripheral nerve sheath tumors (MPNST) and to increase accessibility of these tissues to the research community.","Accessing the data from this study requires a Synapse account. You must also [request access](https://www.synapse.org/#!Synapse:syn4939902/wiki/593715) from the study investigators.",'If you use data from this study in a publication or talk, please cite the following preprint: ""Pollard, K, et al. A clinically and genomically annotated nerve sheath tumor biospecimen repository. bioRxiv 2019.12.19.871897; doi: https://doi.org/10.1101/2019.12.19.871897 .” In addition, please acknowledge the NF Data Portal using the following statement: ""The results published here are in whole or in part based on data obtained from the NF Data Portal (http://www.nf.synapse.org, RRID:SCR_021683) and made available through the NF Open Science Initiative.""',"syn4939902"]},{rowId:502,versionNumber:897,values:["A Robust Plexiform Neurofibroma Model for Preclinical Drug Screening","syn8016635","We utilized a genetically engineered mouse model as a tool to identify the cell of origin for plexiform neurofibroma. Using genetic labeling for cell-lineage tracing, we showed that the embryonic GAP43+PLP+ Schwann cell precursors originate from dorsal spinal nerve roots are the cells of origin for plexiform neurofibromas. Our studies point to the importance of stem cells and their immediate progenitors in the initiation of tumors, consistent with the notion that these neoplasms originate in a subset of primitive precursors and that most cells in an organ do not generate tumors. The identity of the tumor cell of origin and facility for isolation and expansion of these DRG/nerve root neurosphere cells (DNSCs), which contain the embryonic GAP43+PLP+ Schwann cell precursors, allow us to reconstruct all the biological steps from the original cells to tumor stage to delineate mechanism of neurofibromagenesis. Most importantly, the ability to isolate embryonic PLP+ GAP43+ cells for ex vivo expansion and Nf1 ablation to generate classic sciatic plexiform neurofibromas provides a reliable and robust non-germline neurofibroma model for in vivo drug screening with a quantifiable and reproducible outcome measure (via bioluminescent scan) to select lead compounds for clinical assessment. In this application, we propose to establish the utility of this non-germline plexiform neurofibroma (pNF) model as a rapid preclinical therapeutic drug screening tool to identify effective therapies for pNF. To achieve this goal, we will first explore the possibility of long term storage of DNSCs to maintain their tumorigenic potential for reliable and rapid plexiform neurofibromagenesis. Next, we will develop it into a robust plexiform neurofibroma model for preclinical drug screening by testing therapeutic effects of different pharmacological interventions.","The data from this study is currently under embargo. Please contact the principal investigator for access to the data.","The data from this study are still under embargo, therefore, if you have been granted access by the data contributor, you must work with them to determine how to acknowledge your collaboration in any manuscripts that arise. In addition, please acknowledge the NF Data Portal like so: “The results published here are in whole or in part based on data obtained from the NF Data Portal (http://www.nf.synapse.org, RRID:SCR_021683) and made available through the NF Open Science Initiative.”","syn8016635"]},{rowId:503,versionNumber:897,values:["Acceptance and  Commitment Training for Adolescents and  Young Adults with Neurofibromatosis Type  I, Plexiform Neurofibromas, and Chronic  Pain: A Phase III Clinical Trial","syn4939896","NF1-related pain is often not well-controlled with medication and many of the medications cause significant side effects. To our knowledge, only one prior study (by our group) has examined the effectiveness of a psychological intervention for chronic pain in adolescents and young adults (AYA) with NF1 and PNs. Acceptance and Commitment Therapy (ACT) is a type of therapy that focuses on encouraging individuals to engage in more adaptive ways of coping with pain. The goal of ACT is not to eliminate the person's pain, but to optimize the person's functioning despite their pain. This study is examining the impact of an 8-week ACT intervention on pain interference and other variables among AYA ages 16-34 years with NF1, one or more PNs, and chronic pain. We also are exploring the role of heart rate variability (HRV), which is a physiological marker of pain, in this population. The goal of this project is to establish ACT's effectiveness as a psychological intervention tool targeting chronic pain in patients with NF1/PN, in order to improve daily functioning and pain interference. Our primary goal is to compare changes in pain interference scores from pre- to post- intervention between the ACT group and the waitlist (WL) group. Secondary aims include examining baseline to post-treatment changes in quality of life, pain severity, depression, pain-related anxiety, and HRV, assessed by electrocardiogram (ECG); comparing the ACT and WL groups on pre-post changes in quality of life, pain intensity, depression, pain-related anxiety, and HRV; and examining the relationships of HRV with pain and psychosocial variables.",null,"The data from this study are still under embargo, therefore, if you have been granted access by the data contributor, you must work with them to determine how to acknowledge your collaboration in any manuscripts that arise. In addition, please acknowledge the NF Data Portal like so: “The results published here are in whole or in part based on data obtained from the NF Data Portal (http://www.nf.synapse.org, RRID:SCR_021683) and made available through the NF Open Science Initiative.”","syn4939896"]},{rowId:504,versionNumber:897,values:["Can targeted therapy prevent neurofibroma growth in mice?","syn4939872","Preliminary efficacy studies conducted through the NFTC showed that DhhCre;Nf1fl/fl;Stat3fl/fl mice had minimal nerve disruption and, although neurofibromas formed, they were dramatically reduced in size as compared to those in DhhCre;Nf1fl/fl mice. As a result, the hypothesis was formulated that Stat3 inhibition using a Stat3 Antisense RNA will prevent tumor formation in a mouse neurofibroma model. The inhibitor, AZD 9150 / ISIS 481464 showed efficacy in multiple myeloma and is in Phase1/1b trials in other cancers. If this same inhibitor shows efficacy in preventing tumor formation, pre-clinical studies will be executed to assess dosing strategies for limited duration.","N/A","N/A","syn4939872"]},{rowId:505,versionNumber:897,values:["Development of a Child Neurofibromatosis Type 1 Health Related Quality of Life Measure","syn4939878","To develop and test the feasibility, reliability and validity of the child self-report and parent proxy-report versions of the Pediatric Quality of Life InventoryTM (PedsQL-TM) NF1 module for pediatric patients ages 5-25 with NF1 from the perspectives of patients and parents. The PedsQL(TM) NF1 Module Scales demonstrated acceptable to excellent measurement properties, and may be utilized as standardized metrics to assess NF1-specific symptoms and problems in clinical research and practice in children, adolescents, and young adults. The PedsQL(TM) NF1 Module will have a significant impact on the design of clinical trials targeted to better understand and improve clinical treatment for NF1.","The data from this study is currently under embargo. Please contact the principal investigator for access to the data.","The data from this study are still under embargo, therefore, if you have been granted access by the data contributor, you must work with them to determine how to acknowledge your collaboration in any manuscripts that arise. In addition, please acknowledge the NF Data Portal like so: “The results published here are in whole or in part based on data obtained from the NF Data Portal (http://www.nf.synapse.org, RRID:SCR_021683) and made available through the NF Open Science Initiative.”","syn4939878"]},{rowId:506,versionNumber:897,values:["Combination Index Validation Studies (2 drug combinations)","syn4939876","Preliminary High-throughput screening (HTS) data generated at National Center for Advancing Translational Sciences (NCATS) implicated a number of different compounds as potential candidates and combinations suitable for moving forward to preclinical testing, with nine combinations prioritized for validation in murine primary embryonic Schwann cell lines and/or immortalized cell lines. The goal of these experiments was to validate the results of preliminary HTS at NIH/NCATS by Dr. Ferrer.","Accessing the data from this study requires a Synapse account.",'If you use data from this study, please acknowledge the NF Data Portal using the following statement: "The results published here are in whole or in part based on data obtained from the NF Data Portal (http://www.nf.synapse.org, RRID:SCR_021683) and made available through the NF Open Science Initiative."',"syn4939876"]},{rowId:507,versionNumber:897,values:["Cutaneous Neurofibroma - Models, Biology, and Translation","syn11374333","There are still gaps in our knowledge of cNF pathogenesis that could be addressed by the adoption of a systemic approach to decipher all the biological steps in cNF development from the cell of origin to tumor stage to identify the rate limiting step that can be therapeutically targeted to prevent or delay cNF formation. Therefore, in this application, we propose a set specific aims to fill this knowledge gap: We will define the identity of the cNF cell of origin in mice that will guide us to identify the cells of origin for human cNF. We will next decipher how the cell of origin and the tumor microenvironment contribute to drive cNF initiation that will uncover its pathogenesis to delineate biological steps and mechanisms in tumor development as well as generating of preclinical models system that elucidate cNF biology and enable preclinical therapeutic testing. Our overall project will advance via two parallel but interwoven efforts: (1) Characterize cNF cell of origin to trace their developmental origin as well as to define their phenotypes and interactions with the tumor microenvironment. This will provide a framework for elucidating the functional contributions of cNF cell of origin in neurofibroma development. (2) To determine biological steps that are critical for cNF development. A major innovation in our work is linking these approaches in a synergistic manner to our novel neurofibroma models that can be used broadly to decipher the pathogenesis as well as pre-clinical models for therapeutic testing.","Accessing the data from this study requires a Synapse account.","Please acknowledge the NF Data Portal like so: “The results published here are in whole or in part based on data obtained from the NF Data Portal (http://www.nf.synapse.org, RRID:SCR_021683) and made available through the NF Open Science Initiative.”","syn11374333"]},{rowId:508,versionNumber:897,values:["Deconstruction and Reconstitution of NF1 Cutaneous Neurofibromas","syn11374357","Our goal is to substantially increase understanding of the pathophysiology of the cutaneous neurofibromas (cNFs) that are a major burden to many patients with type 1 neurofibromatosis (NF1). We will do this by first deconstructing human cNFs through single-cell transcriptomics (scRNA-Seq) to rigorously define the cell populations and sub-populations in these complex tumors. We will then develop three-dimensional (3D) co-culture models of these tumors as a step toward pathomimetic systems for more accurate preclinical therapeutic screening.The goal of the first aim is to use scRNA-Seq to develop an unbiased portrait of the cellular composition of NF1 cNFs. Each tumor that is deconstructed will be a milestone along the way to reaching that goal. We anticipate that at least 8 tumors will need to be deconstructed to take account of the high degree of heterogeneity in the these tumors. Approximately 10,000 cells from each sample will be subject to scRNA-Seq, with the rest used for development of primary cNF cells and fibroblasts for use in Aim Two. The second aim has the goal of development of pathomimetic avatars of NF1 cNFs that will be useful for therapeutic testing. The number and complexity of the avatars required will be determined by the results of the scRNA-Seq. Milestones will be successful development of 3D culture and co-culture conditions for the cell types and of quantitative assays for their morphology, proliferation, and apoptosis.","The data from this study is currently under embargo. Please contact the principal investigator for access to the data.","The data from this study are still under embargo, therefore, if you have been granted access by the data contributor, you must work with them to determine how to acknowledge your collaboration in any manuscripts that arise. In addition, please acknowledge the NF Data Portal like so: “The results published here are in whole or in part based on data obtained from the NF Data Portal (http://www.nf.synapse.org, RRID:SCR_021683) and made available through the NF Open Science Initiative.”","syn11374357"]},{rowId:509,versionNumber:897,values:["Development of a needs-based quality of life Patient Reported Outcome measure specific to patients with NF1-associated pNFs","syn4939880","The goal of this project is to establish that interventions to NF1-associated pNFs improve/benefit a patient's QoL, in addition to clinical status. The study was designed to develop a patient-based measure that determines the impact of plexiforms on need fulfilment in affected individuals. The model underlying the new measure is that illness and its treatment affect individuals by limiting their ability to meet their human needs. Needs are fundamental to life as they motivate us to take appropriate actions. The work is being conducted in parallel in the US and UK. Consequently, the final scale will be valid for use in both countries individually or combined. A secondary aim of the study is to work with the European Medicines Agency (EMA) to gain favourable Qualification Opinion for the new PRO. This will ensure that data generated from the measure in clinical trials will be accepted by the EMA.","N/A","N/A","syn4939880"]},{rowId:510,versionNumber:897,values:["Development of Patient Reported Outcomes System for Patients with NF1-Associated Plexiform Neurofibromas using Mixed Method Approach","syn4939888","Patient Reported Outcome (PRO) measures assess the treatment outcomes most relevant to patients, and constitute potential outcomes to evaluate in clinical trials. PROs meeting FDA guidelines are critical to ensuring patient-centered endpoints are incorporated. Historically, NF1-associated plexiform neurofibromas (pNF) treatment trials have under-utilized PROs specific to pNF. A psychometrically sound measure incorporating input from patients, families and clinicians, is critical for inclusion in treatment trials. To address the need for a pNF-specific PRO measure for use in clinical trials, this project sought to develop a PRO measurement system utilizing input from patients, families and clinicians, as well as well as psychometric analysis methods to ensure the developed measures are reliable and valid. Specifically this project designed a PRO system for children and young adults with pNF by (1) establishing a conceptual model describing HRQOL, 2) Developing item pools (e.g., pain, stigma, and organ dysfucntion measures), 3) Evaluating psychometric properties of the pools, 4) affirming clinical differences.","Accessing the data from this study requires a Synapse account.","The data from this study are still under embargo, therefore, if you have been granted access by the data contributor, you must work with them to determine how to acknowledge your collaboration in any manuscripts that arise. In addition, please acknowledge the NF Data Portal like so: “The results published here are in whole or in part based on data obtained from the NF Data Portal (http://www.nf.synapse.org, RRID:SCR_021683) and made available through the NF Open Science Initiative.”","syn4939888"]},{rowId:511,versionNumber:897,values:["Drug Screening of pNF Cell Lines","syn4939906","A lack of cellular model systems was identified as a major gap in pNF therapeutics-focused research when NTAP was launched in 2012. Hence, creating a cell culture initiative was an early priority. While current animal models can faithfully mimic some aspects of pNF (tumor histology, imaging characteristics, natural history), they are based on relatively narrow genetically engineered platforms and are not fully representative of human NF1 diversity. Moreover, the tumors in these models are slow growing (requiring 9-12 months for a single compound efficacy study) and costly. Hence, there is a need for in vivo models that allow more rapid drug screening (particularly when considering possible combinatorial therapies) and a need for in vitro models that allow more facile manipulation of the NF1 gene in order to understand the molecular genomics influencing tumor behavior. The goal of this project is to create a panel of cell culture systems that represent pNF complexity and can be utilized to both screen new compounds as well as to identify new therapeutic targets.","Accessing the data from this study requires a Synapse account.",'If you use data from this study in a publication or talk, please cite the following publication: "Ferrer M, Gosline SJC, Stathis M, et al. Pharmacological and genomic profiling of neurofibromatosis type 1 plexiform neurofibroma-derived schwann cells. Sci Data. 2018;5:180106. Published 2018 Jun 12. doi:10.1038/sdata.2018.106." In addition, please acknowledge the NF Data Portal using the following statement: "The results published here are in whole or in part based on data obtained from the NF Data Portal (http://www.nf.synapse.org, RRID:SCR_021683) and made available through the NF Open Science Initiative."',"syn4939906"]},{rowId:512,versionNumber:899,values:["Identification of pathways triggered by Schwann cell-fibroblast interactions driving cutaneous neurofibroma growth","syn11374353","Inactivation of the NF1 gene in SCs is believed to be the cause triggering cNF formation. Extensive analysis has shown that from a genetic and genomic perspective, there is no additional recurrent genetic alteration besides the NF1 inactivation involved in cNFs. Despite its simple genetic etiology, the contribution of the interactions among cells composing cNFs need to be investigated to better understand their role in cNF formation and their potential influence on treatment response. In this work, we propose to use co-cultures of cNF-derived primary SCs and FBs to identify expression signatures in cNFs that are specifically due to SC-FB heterotypic interactions. For this purpose we will compare RNA-Seq data from cNFs, their single cell primary cultures and co-cultures of SCs and FBs. Furthermore, we will identify the genetic variation present in gene coding regions of neurofibroma donors. Since SCs and FBs used in co-cultures will belong to different individuals, we estimate to dissect about 75% of the expression signature due to SC-FB interactions by assigning each differently expressed gene to a specific cell type, according to their genetic variation. Signaling pathways represented by expression signatures will be analyzed and their disruption tested functionally. The goal of this study is to understand the signaling between SCs and FBs within cNFs and their importance and impact on their viability, proliferation capacity and contribution to cNF development.","Accessing the data from this study requires a Synapse account.",null,"syn11374353"]},{rowId:513,versionNumber:897,values:["Genetic Studies of Neurofibromatosis","syn11374339","NF1 cutaneous neurofibromas (cNFs) have some similarities to non-cutaneous (plexiform) neurofibromas, but there are also differences whose molecular bases are not well characterized. There is also heterogeneity within cNFs, e.g. NF1 mutations and phenotypes. To improve our understanding of cNF biology, we will develop characterized cNF Schwann cell two-hit cultures/lines, and perform exome RNA sequence analysis in these resources as well as primary tissue (intact and isolated single cells). Heterozygous Schwann cell lines will also be developed and studied from some of the same tumors.","The data from this study is currently under embargo. Please contact the principal investigator for access to the data.","The data from this study are still under embargo, therefore, if you have been granted access by the data contributor, you must work with them to determine how to acknowledge your collaboration in any manuscripts that arise. In addition, please acknowledge the NF Data Portal like so: “The results published here are in whole or in part based on data obtained from the NF Data Portal (http://www.nf.synapse.org, RRID:SCR_021683) and made available through the NF Open Science Initiative.”","syn11374339"]},{rowId:515,versionNumber:897,values:["Imaging Biomarkers of Neurofibromatosis Type 1-Associated Optic Pathway Glioma","syn4939894","The mechanism of visual acuity loss in optic pathway gliomas (OPG) involves damage to white matter connections due to tumor infiltration or compression of the visual pathway. Non-invasive methods of investigating white matter tract integrity and tissue microstructure may help measure and predict visual acuity in children with OPG. Research supported by the Francis S. Collins Scholarship examines two innovative and non-invasive measures of tissue microstructure: Diffusion Tensor Imaging (DTI) is a magnetic resonance technique to quantify white matter tracts and assess tract damage by measuring the preferential motion of water molecules along and not across hydrophobic myelin. In preliminary data, measures of white matter integrity in the optic radiations of children with OPG are associated with current visual acuity and may predict visual acuity one year later. Magnetic Resonance Fingerprinting (MRF) uses pseudo-randomized magnetic resonance acquisition parameters and a matching algorithm to rapidly quantify T1, T2, M0 and off-resonance signals. These measures of tissue microstructure have been associated with identification of tissue type and tumor grade, but have never before been available so rapidly and reliably. The goal of this project is to interrogate 2 two promising advanced magnetic resonance techniques, DTI and MRF, investigating tissue microstructure in OPG.",null,"The data from this study are still under embargo, therefore, if you have been granted access by the data contributor, you must work with them to determine how to acknowledge your collaboration in any manuscripts that arise. In addition, please acknowledge the NF Data Portal like so: “The results published here are in whole or in part based on data obtained from the NF Data Portal (http://www.nf.synapse.org, RRID:SCR_021683) and made available through the NF Open Science Initiative.”","syn4939894"]},{rowId:516,versionNumber:897,values:["Leveraging human induced pluripotent stem cells (iPSCs) to determine the impact of patient-derived NF1 gene mutations on peripheral sensory neuron-driven Schwann cell growth","syn11374345","Cutaneous or dermal neurofibromas (cNFs) are composed of neoplastic Schwann cells embedded in a microenvironment containing terminal nerve endings, fibroblasts, mast cells, and macrophages. While caused by loss of NF1 protein function in Schwannian lineage cells, studies in mouse plexiform neurofibroma models have revealed a critical role for non-neoplastic stromal cells (fibroblasts, mast cells, and macrophages) in tumor formation, maintenance, and continued growth. In this regard, inhibition of stromal cell (mast cells, macrophages) function in preclinical genetically-engineered mice resulted in attenuated plexiform neurofibroma growth. In addition to fibroblasts, mast cells, and macrophages, the NF1-deficient Schwann cells in cNFs often grow in proximity to terminal nerve endings. The intimate relationship between neurons and Schwann cells during normal peripheral nervous system development and function raises the intriguing possibility that neurons are important drivers of cNF growth. This notion is supported by recent studies in which the proliferation of myelinating cells in the central nervous system (oligodendrocytes) is regulated by neuronal production of a novel growth factor (neuroligin-3). Based on these provocative findings, we hypothesize that peripheral sensory neurons accelerate NF1-deficient Schwann cell growth through the elaboration of mitogens. To identify these potentially clinically-actionable mitogens, we have designed a series of milestone-driven experiments using a novel collection of patient-derived and isogenic human induced pluripotent stem cells harboring NF1-patient germline NF1 gene mutations.","Accessing the data from this study requires a Synapse account.",'If you use data from this study in a publication or talk, please cite study here:  Anastasaki, C., Mo, J., Chen, JK. et al. Neuronal hyperexcitability drives central and peripheral nervous system tumor progression in models of neurofibromatosis-1. Nat Commun 13, 2785 (2022). https://doi.org/10.1038/s41467-022-30466-6. In addition, please acknowledge the NF Data Portal using the following statement: "The results published here are in whole or in part based on data obtained from the NF Data Portal (http://www.nf.synapse.org, RRID:SCR_021683) and made available through the NF Open Science Initiative.',"syn11374345"]},{rowId:517,versionNumber:897,values:["Microenvironmental targets of cutaneous neurofibromas: T-cells and mast cells as tumor contributors","syn11374341","Immunity has a central and complex role in tumor growth, which can be utilized when developing new treatment modalities for tumors. To date, the role of immune system in cNFs has gained little attention. Transformed cells are normally eradicated by cooperation of innate and adaptive immune cells. Immune surveillance could be expected to recognize neurofibromin neoepitopes resulting from the second hit of the NF1 gene, but this is not taking place in cNFs. Mast cells are known to be abundant in cNFs and they are increasingly more interesting as druggable targets in tumor biology and neurogenic itch. It is also not known whether mast cells of cNFs are tumorigenic or anti- tumorigenic. The goal of this project is to prove that T cells and mast cells are valid targets for the treatment of cNF. Elucidating the characteristics, roles and effects of these prominent tumor microenvironment components in promoting tumorigenesis could be used to facilitate finding of targeted therapies for the treatment of cNF. The therapeutic modalities include those already approved and those being developed for tumor biology as well as for inflammatory diseases.","A subset of this project has been released: www.doi.org/10.7303/syn25613686.The data from this study is currently under embargo. Please contact the principal investigator for access to the data.",'If you use data from this study in a publication or talk, please cite the following publication: "Kallionpää R, A, Ahramo K, Martikkala E, Fazeli E, Haapaniemi P, Rokka A, Leivo I, Harvima I, T, Peltonen J, Peltonen S: Mast Cells in Human Cutaneous Neurofibromas: Density, Subtypes, and Association with Clinical Features in Neurofibromatosis 1. Dermatology 2021. doi: 10.1159/000517011." In addition, please acknowledge the NF Data Portal using the following statement: "The results published here are in whole or in part based on data obtained from the NF Data Portal (http://www.nf.synapse.org, RRID:SCR_021683) and made available through the NF Open Science Initiative."',"syn11374341"]},{rowId:518,versionNumber:897,values:["Modeling signaling networks of NF1-deficient Schwann cells and plexiform neurofibromas using mass spectrometry-based proteomics","syn8016599","We aim to identify putative proteomic and/or phosphoproteomic signatures associated with loss of NF1. Recent advances in mass-spectrometry (MS)-based proteomics permit the determination of the extent, localization, and site-specific stoichiometry of protein phosphorylation. We propose using two complementary MS-based approaches to define (i) the proteome and phosphoproteome and (ii) the activated kinome signature of NF1-deficient Schwann cells (SCs). We will perform analyses on a panel of SC lines derived from PNs. Transcriptome profiling of cell lines will complement these data sets, together providing a detailed view of the cellular signaling networks and the molecular targets controlled by neurofibromin in a cell-type relevant to the disease. Finally, we will use small molecule inhibitors to both known and any novel signaling pathways we identify to examine the adaptive kinome in treated cell lines. Combining the use of a novel panel of cell lines with the sensitivity of high-end MS, we expect to identify signaling pathways and targets previously implicated as being important in NF1-associated tumors, as well as new and unexpected targets downstream of NF1.",null,"Please acknowledge the NF Data Portal like so: “The results published here are in whole or in part based on data obtained from the NF Data Portal (http://www.nf.synapse.org, RRID:SCR_021683) and made available through the NF Open Science Initiative.”","syn8016599"]},{rowId:519,versionNumber:897,values:["Identification of Neurofibroma Growth and Drug Resistance Pathways","syn4939892","Up to 50%> of children with NF1 develop plexiform neurofibromas that can cause severe disfigurement and serious morbidity-- and death when they compress vital organs. Most plexiform neurofibromas are resistant to chemotherapy and surgical removal is often impossible due to tumor size, location, and nerve integration. We jo not know why some human neurofibromas grow slowly and some rapidly, nor why some neurofibromas respond to therapy but others do not. Plexiform neurofibromas grow most aggressively in young children, and an appealing (unproven) idea is that transiently blocking growth of plexiform neurofibromas early in tumor development will prevent tumor progression. Models to study these critical issues have been lacking. Our laboratory has recently developed mouse models that recapitulate 1) slow and fast growing neurofibromas, and 2) neurofibromas that respond or fail to respond to targeted molecular therapy. We plan to use new methods in gene expression analysis to identify molecular pathways that correlate with these phenotypes in mouse models, with verification on human tissues. We also plan test if early treatment delays or halts neurofibroma formation by administering drugs before tumors develop in a neurofibroma mouse model.",null,"The data from this study are still under embargo, therefore, if you have been granted access by the data contributor, you must work with them to determine how to acknowledge your collaboration in any manuscripts that arise. In addition, please acknowledge the NF Data Portal like so: “The results published here are in whole or in part based on data obtained from the NF Data Portal (http://www.nf.synapse.org, RRID:SCR_021683) and made available through the NF Open Science Initiative.”","syn4939892"]},{rowId:520,versionNumber:897,values:["Neurofibromatosis Type 1 Dermal Neurofibroma Longitudinal Natural History Study and Selumetinib Clinical Trial","syn8012530","Previous reports have correlated increased burden of dermal neurofibromas (DNs) with age and pregnancy, but longitudinal data are not available to establish a quantitative natural history of these tumors. Moreover, the genomic mechanisms responsible for dermal tumor proliferation are not completely understood, although biallelic inactivation of the NF1 gene is known to be required. The NF1 gene encodes neurofibromin, which negatively regulates the Ras/MAPK signaling cascade that includes the RAF/MEK/ERK and PI3K/AKT/mTOR pro-growth pathways. Current therapeutic development for NF1-related tumors target components of this pathway; however, clinical trials directed at dermal neurofibromas have been hampered by the difficulty in measuring and counting dermal neurofibromas. The overall goal of this project is to complete a longitudinal natural history study of dermal neurofibromas to enhance current knowledge of dermal neurofibroma development as well as improve the design, implementation, and interpretation of clinical trials targeting DNs. To address the clinical trial facet of this project, a trial using the MEK1/2 inhibitor selumetinib will be piloted. We will also evaluate the quality of life impact of dermal neurofibromas before and during Selumetinib treatment. For both facets of this study, we will examine genomic alterations that correlate with DN development and selumetinib response.","The data from this study is currently under embargo. Please contact the principal investigator for access to the data.","The data from this study are still under embargo, therefore, if you have been granted access by the data contributor, you must work with them to determine how to acknowledge your collaboration in any manuscripts that arise. In addition, please acknowledge the NF Data Portal like so: “The results published here are in whole or in part based on data obtained from the NF Data Portal (http://www.nf.synapse.org, RRID:SCR_021683) and made available through the NF Open Science Initiative.”","syn8012530"]},{rowId:521,versionNumber:897,values:["Cutaneous Neurofibroma Data Resource","syn4984604","In 2013, CTF began a neurofibromatosis biobank to collect NF samples to better study the disease. With all samples being collected under the same protocol, the biobank enables comprehensive analysis of diverse NF samples. To showcase the value the biobank, CTF has kicked off a pilot project designed to characterize the molecular landscape of cutaneous neurofibromas (NF) that grow in NF1 patients, sponsored by the Children's Tumor Foundation. This project includes high-throughput data from 11 different patient tumors and matched blood samples. These data are then compared with one another to assess the distinct landscape of cutaneous NF for each patient.","Accessing the data from this study requires a Synapse account. You must also [request access](https://www.synapse.org/#!Synapse:syn4984604/wiki/594099) from the Children's Tumor Foundation.",'If you use data from this study in a publication or talk, please cite the following publication: "Gosline SJ� Weinberg H, Knight P, et al. A high-throughput molecular data resource for cutaneous neurofibromas. Sci Data. 2017;4:170045. Published 2017 Apr 11. doi:10.1038/sdata.2017.45." In addition, please acknowledge the NF Data Portal using the following statement: "The results published here are in whole or in part based on data obtained from the NF Data Portal (http://www.nf.synapse.org, RRID:SCR_021683) and made available through the NF Open Science Initiative."',"syn4984604"]},{rowId:522,versionNumber:897,values:["Synodos NF1 Preclinical Models (Iowa Sanford Exemplar)","syn5610446","The aim of Synodos for NF1 is to find solutions to the unique problems that affect NF1 patients in the fastest, most efficient manner by sharing their real-time data and results. In Project 2 of the Synodos for NF1 Consortium, researchers will develop and perfect animal models that will most accurately replicate human reaction to drugs in preclinical testing. Two teams will independently generate models; this project consists of results from the Iowa team, and includes members from the University of Iowa, University of Arizona, Exemplar Genetics and Sanford Health.",'Accessing animal model data from this project requires a Synapse account. <br><br>  Accessing human biospecimen genomics data from this project requires submission of an Intended Data Use Statement and agreement to Data Access Terms of Use.  <br><br> To access data from this project, please follow these steps:  <br><br> **1**: Register for a Synapse account [here](https://www.synapse.org/#!RegisterAccount:0) (if you do not already have one). <br> **2**: Navigate to the [data files page](https://www.synapse.org/#!Synapse:syn11898002). <br> **3**: Click on "Request Access" next to the "lock" symbol. <br> **4**: Follow the instructions on that page. You will be prompted to "Request access to Data" by clicking the Request Access button and submitting an Intended Data Use statement (1-3 paragraphs in English) to be posted publicly. You will also need to review the Terms of Use and click �Accept Terms of Use". If you have questions about the data access process please post them to the [discussion forum](https://www.synapse.org/#!Synapse:syn5610418/discussion/default). <br>','The Data Contributors encourage project results to be made publicly available according to the following terms: <br/> <br/>  You will provide any results that are to be publicly disclosed to the Data Contributor for review according to the following timelines: Manuscripts 30 days in advance and abstracts 10 days in advance. <em>The Data Contributor may request in writing that the proposed publication/disclosure be delayed for up to 30 additional days as necessary to protect proprietary information.</em> **Please email materials for review to benjamin-darbro@uiowa.edu**<br/> <br/> You will acknowledge the NF Data Portal using the following statement: "The results published here are in whole or in part based on data obtained from the NF Data Portal (http://www.nf.synapse.org, RRID:SCR_021683) and made available through the NF Open Science Initiative."  <br/> <br/> If you use the transcriptomic data provided by the **Synodos NF1 Preclinical Models (Iowa Sanford Exemplar) Study** in a publication or talk, you will cite the following paper: <em>Kohlmeyer JL, Kaemmer CA, Pulliam C, et al. RABL6A Is an Essential Driver of MPNSTs that Negatively Regulates the RB1 Pathway and Sensitizes Tumor Cells to CDK4/6 Inhibitors. Clin Cancer Res. 2020. https://doi.org/10.1158/1078-0432.ccr-19-2706</em>',"syn5610446"]},{rowId:523,versionNumber:897,values:["Synodos NF1 Preclinical Models (Minnesota CCHMC Recombinetics)","syn5610425","The aim of Synodos for NF1 is to find solutions to the unique problems that affect NF1 patients in the fastest, most efficient manner by sharing their real-time data and results. In Project 2 of the Synodos for NF1 Consortium, researchers will develop and perfect animal models that will most accurately replicate human reaction to drugs in preclinical testing. Two teams will independently generate models; this project consists of results from the Minnesota team, and includes members from the University of Minnesota, Cincinnati Children's Hospital, the National Cancer Institute, and Recombinetics Inc.","Accessing the data from this study requires a Synapse account.","The data from this study are still under embargo, therefore, if you have been granted access by the data contributor, you must work with them to determine how to acknowledge your collaboration in any manuscripts that arise. In addition, please acknowledge the NF Data Portal like so: “The results published here are in whole or in part based on data obtained from the NF Data Portal (http://www.nf.synapse.org, RRID:SCR_021683) and made available through the NF Open Science Initiative.”","syn5610425"]},{rowId:528,versionNumber:897,values:["Synodos NF1 Low Grade Glioma","syn5698493","Low-grade gliomas (LGGs) are brain tumors that affect children with NF1. Despite their prevalence, there are few effective treatment options for patients with LGG. The goal of the Synodos for NF1 - Low Grade Glioma projects is to improve our understanding of the molecular underpinnings of LGGs, and identify new drug targets that could lead to better treatments for this tumor type.",'Accessing data from this project requires submission of an Intended Data Use Statement and agreement to Data Access Terms of Use.  To access data from this project, please follow these steps: <br> 1: Register for a Synapse account here (if you do not already have one). <br> 2: Navigate to the [data files page](https://www.synapse.org/#!Synapse:syn10145839). <br> 3: Click on "Request Access" next to the "lock" symbol. <br> 4: Follow the instructions on that page. You will be prompted to "Request access to Data" by clicking the Request Access button and submitting an Intended Data Use statement (1-3 paragraphs in English) to be posted publicly. You will also need to review the Terms of Use and click “Accept Terms of Use". If you have questions about the data access process please email nf-osi@sagebionetworks.org.',`In using this data, you promise to acknowledge the data contributors by including the following statement: “The research team would like to thank the Children's Tumor Foundation Synodos Low Grade Glioma team that collected and shared the data used in this study.” Please include the following citation for these data: “Fisher, M.J., Jones, D.T.W., Li, Y. et al. Integrated molecular and clinical analysis of low-grade gliomas in children with neurofibromatosis type 1 (NF1). Acta Neuropathol (2021). https://doi.org/10.1007/s00401-021-02276-5" In addition, please acknowledge the NF Data Portal like so: “The results published here are in whole or in part based on data obtained from the NF Data Portal (http://www.nf.synapse.org, RRID:SCR_021683) and made available through the NF Open Science Initiative.`,"syn5698493"]},{rowId:529,versionNumber:897,values:["Columbia University NF1 Glioma Project","syn6633069","This study aims to obtain the genomic landscape of glioma associated with NF1 syndrome. While the clinical aspects of NF1 are well characterized and the pathogenesis of plexiform neurofibromas has been widely studied, surprisingly little is known about the genetic and epigenetic changes in the NF1 mutant glioma. This has hindered the development of effective therapies for NF1-mutant cancers. To address this challenge and uncover the driver genes in NF1-glioma, The team will analyze whole-exome and transcriptome data from glioma samples of pediatric and adult NF1 patients. Alterations in the genetic networks will be contextualized in the natural genetic and cellular environment of a specific tumor, and the key driving modules on which specific NF1-glioma subgroups rely for growth, survival and progression will be identified. With this information in hand, we can target the critical alterations with specific drugs, often already available for other types of diseases. The specific aims are to (1) comprehensively characterize the genomic and epigenetic landscape of NF1 gliomas from patients of all age groups, (2) identify driver genetic alterations in NF1-gliomas from the integration of somatic copy number variations, mutations and gene fusions (translocations), (3) reconstruct a somatic alteration network for the inference of mutual exclusivity and co-occurrence of genetic aberrations in NF1-gliomas and (4) validate the functional significance of the inferred alterations in neural systems in vitro and in vivo, and target therapeutically key genetic nodes with highly specific inhibitors.","Accessing the data from this study requires a Synapse account.",`If you use data from this study in a publication or talk, please cite the following publication: "D'Angelo, F., Ceccarelli, M., Tala et al. The molecular landscape of glioma in patients with Neurofibromatosis 1. Nat Med 25, 176-187 (2019). https://doi.org/10.1038/s41591-018-0263-8" In addition, please acknowledge the NF Data Portal using the following statement: "The results published here are in whole or in part based on data obtained from the NF Data Portal (http://www.nf.synapse.org, RRID:SCR_021683) and made available through the NF Open Science Initiative.`,"syn6633069"]},{rowId:530,versionNumber:897,values:["Development of a Novel Plexiform Neurofibroma Farming System for 96-well Plate Drug Screening","syn7217946","Plexiform neurofibromas (PNFs) are relatively slow growing benign tumors that are difficult to maintain in culture over long passages. As benign tumors, PNFs are likely to be more dependent on factors in the microenvironment, such as the extracellular matrix (ECM) and paracrine growth factors, compared to malignant tumors. It is now clear that many cell types grow better in three-dimensional cultures than on two-dimensional growth surfaces, and it may be that PNFs will growth more robustly in a system that mimics their in vivo growth patterns. The goal of this project is to determine the optimum ECM substrate for nf1 null Schwann cell growth, and to develop a pNF culture system to be used in 96-well based assays (preclinical screening applications). The main goal is to improve the culture conditions for PNF cells, so that these benign tumors can be more reliably cultured from primary tumors. Our secondary goal is to develop culture conditions that lead to PNF gene expression that most closely mimics PNF cells in primary tumors. By developing faithful culture conditions, PNF cells can be tested in drug screen assays with more accurate cell responses to compounds being tested.","N/A","N/A","syn7217946"]},{rowId:531,versionNumber:898,values:["Novel therapeutic development of NF1-associated malignant peripheral nerve sheath tumor (MPNST)","syn11681835","Malignant Peripheral Nerve Sheath Tumor (MPNST) arising from the cellular component of the peripheral nerves represents a group of highly aggressive soft tissue sarcomas with limited treatment options. They can occur in the setting of type I neurofibromatosis (NF1-associated), or sporadically or in association with prior radiotherapy (RT-associated). Using comprehensive genomic approaches, we and others have discovered that the majority of MPNSTs have inactivation of three tumor suppressor pathways, including the NF1, the CDKN2A (INK4A/ARF) and the Polycomb repressive complex 2 (PRC2) pathways. Here, we propose to use multidisciplinary approaches to establish authentic MPNST model systems for preclinical investigation. We will also evaluate a combination treatment strategy of trametinib (a MEK inhibitor) and ribociclib (a CDK4/6 inhibitor) in various cell line and animal models of MPNST. Lastly, we will conduct a screen using RNA interference technology to identify and characterize the genes that are important for MPNST growth and survival when the PRC2 pathway is inactivated. These studies will build important resources for the NF1-research community and can identify novel treatment strategies and streamline clinical therapeutic development for patients with NF1-associated MPNSTs.","The data from this study is currently under embargo. Please contact the principal investigator for access to the data.","The data from this study are still under embargo, therefore, if you have been granted access by the data contributor, you must work with them to determine how to acknowledge your collaboration in any manuscripts that arise. In addition, please acknowledge the NF Data Portal like so: “The results published here are in whole or in part based on data obtained from the NF Data Portal (http://www.nf.synapse.org, RRID:SCR_021683) and made available through the NF Open Science Initiative.”","syn11681835"]},{rowId:532,versionNumber:897,values:["Development of a novel treatment approach to Neurofibromatosis Type 1-related MPNST using genetically engineered Clostridium novyi-NT","syn4939884",`Due to the invasive nature and high incidence of metastases of malignant peripheral nerve sheath tumors (MPNSTs), surgery is the only treatment modality proven to have survival benefit and even when maximal surgery is feasible, these tumors are almost never curable. MPNSTs develop tumor hypoxia and necrosis as a pathophysiologic consequence of structurally and functionally disturbed microcirculation. Although tumor hypoxia is strongly associated with tumor propagation and treatment resistance, it also offers the perfect niche for the growth of the ""therapeutic"" anaerobic spore-forming bacterium Clostridium novyi-NT (C. novyi-NT). C. novyi-NT's oncolytic potential has been successfully used to treat a variety of solid cancers. Once injected, C. novyi-NT spores germinate in tumor hypoxia while vegetative bacteria cannot survive in normoxia and sporulate. We recently demonstrated robust and reproducible antitumor responses when C. novyi-NT spores were injected into dogs with spontaneous peripheral nerve sheath tumors and in human patients with therapy resistant sarcomas (NCT01924689). The goal of this project is to explore C. novyi-NT as a novel therapeutic for NF1-associated MPNSTs (gather preclinical evidence substantiating clinical trial for for C. novyi-NT therapy in NF1 patients with MPNSTs)`,"N/A","N/A","syn4939884"]},{rowId:533,versionNumber:897,values:["Exploring the Plexiform Neurofibroma Interactome","syn4939890","NF1 mutations in Schwann cells (SCs) drive neurofibroma formation; thus the primary neoplastic cell type in neurofibromas is derived from the SC lineage. The SC genes that are deregulated in mouse and human neurofibromas have not been fully identified. Many cell types are present in neurofibromas besides SCs. These include fibroblasts, mast cells, macrophages, and endothelial cells that are NF1 wild type (as in solitary neurofibroma or in somatic mosaic patients) or NF1 heterozygous (as in many NF1 patients and some mouse models). But how these cells and the intercellular communications contribute to plexiform neurofibroma growth is unknown. It has been recently identified that macrophage is a major cell type in neurofibromas. Macrophages shift gene expression and function in response to the environment, including after nerve injury. Macrophages show anti- tumor effects by secreting free-radicals and recruiting the adaptive immune system (primarily T-cells). However as tumors progress, macrophages stimulate tumor growth and blood vessel formation. Injury-associated inflammation facilitates neurofibroma development by recruiting macrophages in mouse models. How these inter- and intra-cellular interactions contribute to neurofibroma formation has not been studied yet. The goals of this project are to characterize gene expression profiles of Schwann cells and macrophages in mouse nerves and neurofibromas, identify cytokine/chemokine pathways that slow or promote tumor growth and identify molecular networks that mediate intra- and inter-cellular signaling in controlling neurofibroma growth.","Accessing the data from this study requires a Synapse account.",'If you use data from this study in a publication or talk, please cite the following publication: "Choi K, Komurov K, Fletcher JS, Jousma E, Cancelas JA, Wu J, Ratner N. An inflammatory gene signature distinguishes neurofibroma Schwann cells and macrophages from cells in the normal peripheral nervous system. Scientific Reports 2017 Mar; 7, 43315. DOI:10.1038/srep43315." In addition, please acknowledge the NF Data Portal using the following statement: "The results published here are in whole or in part based on data obtained from the NF Data Portal (http://www.nf.synapse.org, RRID:SCR_021683) and made available through the NF Open Science Initiative."',"syn4939890"]},{rowId:534,versionNumber:964,values:["Plexiform Neurofibroma Model Systems for Preclinical Drug Screening","syn7217928","The high vascularity of plexiform neurofibromas (PNs) can cause major problems during surgery. Most in vitro drug testing is still performed on cells growing in two dimensions (2D) on plastic dishes. In comparison to 2D cultures, cells growing in a 3-dimensional (3D) matrix that models the (patho) physiological environment typically: 1) adopt a morphology that better recapitulates that in vivo; 2) exhibit slower growth more akin to those in vivo; and 3) demonstrate drug sensitivity and resistance that may better reflect potential therapeutic response. Our goals were to develop and optimize robust 3D co-culture models of NF1 PNs to allow preclinical drug screening that is both much more representative of likely translational effectiveness than provided by testing in 2D mono-culture, and at a much higher throughput than is available in animal models. Further, we predict that programs such as the NF Clinical Trials Consortium are more likely to be successful if they build on data from robust and representative pre-clinical models. Our aims were therefore to: 1. Develop, characterize and optimize 3D co-culture models to include the cell types found in NF1 PNs; and 2. Test potential drugs in the 3D models of NF1 PNs, with a focus on candidates chosen for translational potential.","Accessing the data from this study requires a Synapse account.",'If you use data from this study in a publication or talk, please cite the following publication: "Development of 3D culture models of plexiform neurofibroma and initial application for phenotypic characterization and drug screening"  https://doi.org/10.1016/j.expneurol.2017.10.012. In addition, please acknowledge the NF Data Portal like so: “The results published here are in whole or in part based on data obtained from the NF Data Portal (http://www.nf.synapse.org, RRID:SCR_021683) and made available through the NF Open Science Initiative.',"syn7217928"]},{rowId:535,versionNumber:897,values:["Evotec Drug Target Research","syn7239560","A drug-target database was built by Evotec using publicly available data in order to define targets of the Minnesota Synodos NF1 screening molecules. A similar resource can be found in the Drug-Target Explorer project conducted by Sage Bionetworks.","Accessing the data from this study requires a Synapse account.","The data from this study are still under embargo, therefore, if you have been granted access by the data contributor, you must work with them to determine how to acknowledge your collaboration in any manuscripts that arise. In addition, please acknowledge the NF Data Portal like so: “The results published here are in whole or in part based on data obtained from the NF Data Portal (http://www.nf.synapse.org, RRID:SCR_021683) and made available through the NF Open Science Initiative.”","syn7239560"]},{rowId:536,versionNumber:897,values:["A primary plexiform neurofibroma cell culture model for use in cell-based high-throughput screens","syn7217935","A cardinal feature of NF1 is the development of dermal (DN) and plexiform neurofibromas (PN), benign tumors that form in the dermis or along a nerve plexus, respectively. While these tumors are histologically similar - both are complex tumors containing a number of cell types including proliferating Schwann lineage cells (from which the tumors arise), fibroblasts, and infiltrating mast cells - they arise in different locations of the body and at different developmental time points. Although initially benign, PN can undergo malignant transformation and develop into MPNSTs. The only treatment for PN is surgery, however complete resection is often difficult due to nerve involvement. The goal of this project is to generate a physiologically relevant PN cellular model system for use in preclinical drug screening.","N/A","N/A","syn7217935"]},{rowId:537,versionNumber:897,values:["pNF Cell Line Characterization Studies","syn4939874","A lack of cellular model systems was recently identified as a major gap in pNF research that was slowing therapeutic discovery by relying on costly, time-consuming animal systems to identify agents that might be effective against these tumors. When NTAP was initiated, there were 2 cell culture systems for pNF from a single lab that had not been fully characterized and were not deemed suitable for drug discovery. As a direct result of NTAP's efforts, there are now 6 cell culture systems available that are actively used in the pipeline of drug discovery. These cultures are intended to model the importance of the micro-environment in pNF and the heterogeneity of these tumors. The goal of this project is the genetic characterization of NF1-/-, NF1-/+, and NF1+/+ cell-lines to be available as a tool-set for the community.","Accessing the data from this study requires a Synapse account.",'If you use data from this study in a publication or talk, please cite the following publication: "Ferrer M, Gosline SJC, Stathis M, et al. Pharmacological and genomic profiling of neurofibromatosis type 1 plexiform neurofibroma-derived schwann cells. Sci Data. 2018;5:180106. Published 2018 Jun 12. doi:10.1038/sdata.2018.106." In addition, please acknowledge the NF Data Portal using the following statement: "The results published here are in whole or in part based on data obtained from the NF Data Portal (http://www.nf.synapse.org, RRID:SCR_021683) and made available through the NF Open Science Initiative."',"syn4939874"]},{rowId:538,versionNumber:897,values:["Development of Plexiform Neurofibroma Cellular Assay for High-Throughput Screening","syn7239479","There is a need for effective therapies for neurofibromas, particularly the larger, plexiform neurofibromas. Tissue culture is an important tool for research. However, it is difficult to derive enriched human Schwann cell cultures, and most enter replicative senescence after 6-10 passages, impeding cell-based research in NF1. We are well-positioned to develop cellular models, and collaborate with a robotics R&D/Lead Optimization group. THe goals of this project is to identify (create/characterize) semi-immortalized pNF and normal/heterozygous SC lines with suitable phenotypes, such that these lines can be tested in and transitioned to an HTS environment.",null,null,"syn7239479"]},{rowId:539,versionNumber:897,values:["Differentiation of NF1 heterozygous and NF1 homozygous iPS cells into cells of the Neural Crest-Schwann cell lineage","syn4939918","In order to identify effective drugs against plexiform neurofibroma (PNF) formation/ progression we need to test them in the right in vitro and in vivo model systems. This involves, among other things, being able to use large quantities of NF1(+/-) and (-/-) cells closely resembling cells driving PNF formation and growth. PNF-derived NF1(+/-) and (-/-) iPS cells have been generated and can be used to test differentiation protocols towards the Neural Crest (NC) -Schwann cell (SC) lineage. Differentiated cells at different points of this lineage commitment can be characterized and engrafted into the sciatic nerve of nude mice to test their tumor-formation capacity and, in this way, interrogate about the identity of the cells originating PNFs. Two things, the setting up the right in vitro iPSC differentiation conditions towards NC and SC in one side and on the other, the characterization of the different steps of differentiation in the NC-SC lineage, will proportionate the basis for performing future drug screenings for PNFs using iPSC-derived cells. The goals of this project are to (1) set up in vitro conditions to efficiently differentiate iPS cells into cells of the Neural Crest -Schwann cell lineage. Characterize the methylome and transcriptome of these cells with distinct NF1 genotypes and test their potential engraftment/tumor formation capacity. (2) Generate new iPSC and edit control iPSC with distinct NF1 genotypes to interrogate the presence of epigenetic memory in the PNF-derived iPS cells generated","The data from this study is currently under embargo. Please contact the principal investigator for access to the data.","The data from this study are still under embargo, therefore, if you have been granted access by the data contributor, you must work with them to determine how to acknowledge your collaboration in any manuscripts that arise. In addition, please acknowledge the NF Data Portal like so: “The results published here are in whole or in part based on data obtained from the NF Data Portal (http://www.nf.synapse.org, RRID:SCR_021683) and made available through the NF Open Science Initiative.”","syn4939918"]},{rowId:540,versionNumber:897,values:["Perpetuating NF1 heterozygous and NF1 homozygous negative plexiform neurofibroma-derived tumor cells through the generation of induced pluripotent stem (iPS) cells","syn7231973","Plexiform neurofibromas (pNFs) differ from discrete neurofibromas (dNFs) in their moment of appearance (though to be congenital in pNFs), the importance of the extension of the nerve and tissues affected (larger in pNFs) and their potential to progress to malignancy, not present in dNFs. Regarding genomics and transcriptomics, Schwann cells (SC) derived from dNFs and pNFs are very much alike. So what is different among these tumors that confer pNFs a risk to become malignant? The extent of affected tissue? The location? The temporal window at which a neural crest-derived cell committed to the SC lineage acquires a double inactivation of the NF1 gene? A particular cellular composition? A combination of factors? The goals of this project are to (1) Explore the degree of genomic, molecular and cellular variation within a whole plexiform neurofibroma (pNF) and (2) Obtain, authenticate and characterize NF1-/- and NF1+/- induced pluripotent stem cells (iPSC) derived from pNFs.","Accessing the data from this study requires a Synapse account. You must also [request access](https://www.synapse.org/#!Synapse:syn4939902/wiki/593715) from the study investigators.",'If you use data from this study in a publication or talk, please cite the following publication: ""Carrió M, Mazuelas H, Richaud-Patin Y, et al. Reprogramming Captures the Genetic and Tumorigenic Properties of Neurofibromatosis Type 1 Plexiform Neurofibromas [published correction appears in Stem Cell Reports. 2019 Mar 5;12(3):639-641]. Stem Cell Reports. 2019;12(2):411–426. doi: 10.1016/j.stemcr.2019.01.001"" In addition, please acknowledge the NF Data Portal using the following statement: ""The results published here are in whole or in part based on data obtained from the NF Data Portal (http://www.nf.synapse.org, RRID:SCR_021683) and made available through the NF Open Science Initiative',"syn7231973"]},{rowId:541,versionNumber:897,values:["Development of a Preclinical NF1-MPNST Platform Suitable for Precision Oncology Drug Discovery and Evaluation","syn11638893","One of the most common malignancies affecting adults with neurofibromatosis type 1 (NF1) are malignant peripheral nerve sheath tumors (MPNSTs), highly aggressive sarcomas that typically develop from benign plexiform neurofibromas. 8-13% of individuals with NF1 will develop MPNSTs during young adulthood. There are few therapeutic options, and the vast majority of people with these cancers will die within 5 years of diagnosis. Despite advances in our understanding of the pathobiology of these tumors and the identification of seemingly-promising therapeutic targets using a single mouse MPNST model system (NPCis mice and derivative cell lines), no investigational agents have demonstrated efficacy following translation to human clinical trials. However, there is limited genetic diversity being modeled in these systems, which does not capture the full spectrum of genetic diversity that characterizes human NF1-MPNSTs. In this regard, p53 loss, a common feature of the currently-used preclinical models, is observed in a minority of human MPNSTs. We hypothesize that one of the major reasons that rationally-chosen drugs underperform in human clinical trials is that the preclinical models used to discover and evaluate promising therapeutic agents do not capture the genetic heterogeneity of the human cancer. To optimize clinical translation, we aim to develop a collection of MPNST Patient Derived Xenografts (MPNST-PDXs) for the NF community, which more fully represent the spectrum of genetic heterogeneity seen in the human condition. The main goal of this research project is to develop a set of NF1-MPNST PDX lines that captures the spectrum of molecular heterogeneity of the human disease. The proposal aims to meet the following objectives: a. Create enabling resources including 10 NF1-MPNST PDX lines for preclinical studies. b. Histologically and genomically characterize these 10 NF1-MPNST PDX lines, thereby defining the molecular heterogeneity of these tumors. c. Determine whether there is a correlation between molecular subtype and response to chemotherapy or targeted therapy. d. Assess potential adaptive mechanisms or tumor resistance by assessing genomic changes following treatment.","Accessing data from this project requires submission of an Intended Data Use Statement and agreement to Data Access Terms of Use.  <br><br> To request access to data from this project, please follow the steps outlined [here](https://www.synapse.org/#!Synapse:syn11638893/wiki/608144).",'Please cite the following work: "Dehner C, et al. Chromosome 8 gain is associated with high-grade transformation in MPNST. JCI Insight. 2021 Mar 22;6(6):e146351. doi: 10.1172/jci.insight.146351.” <br> Please acknowledge the NF Data Portal using the following statement: "The results published here are in whole or in part based on data obtained from the NF Data Portal (http://www.nf.synapse.org, RRID:SCR_021683) and made available through the NF Open Science Initiative."',"syn11638893"]},{rowId:542,versionNumber:897,values:["Synodos Schwannomatosis","syn9727752","Synodos for Schwannomatosis is a two-year initiative that will bring together basic, translational, and clinical researchers towards the end goal of developing effective treatments for schwannomatosis. The project aims to perform an extensive molecular analysis of schwannomatosis tumor samples to identify new therapeutic targets, and advance the understanding and management of the disease, with a focus on pain. Experts from institutions spread across four different countries are working together to 1) develop a clinically annotated schwannomatosis biobank; 2) develop a comprehensive pain registry for schwannomatosis patients; 3) establish the genomic and epigenomic landscape of schwannomatosis to identify disease-driving alterations; and 4) correlate the molecular findings with key clinical parameters, including pain.",'Accessing data from this project requires submission of an Intended Data Use Statement and agreement to Data Access Terms of Use. <br><br> To access data from this project, please follow these steps: <br><br> **1**: Register for a Synapse account [here](https://www.synapse.org/#!RegisterAccount:0) (if you do not already have one). <br> **2**: Navigate to the [data files page](https://www.synapse.org/#!Synapse:syn17262902). <br> **3**: Click on "Request Access" next to the "lock" symbol. <br> **4**: Follow the instructions on that page. You will be prompted to "Request access to Data" by clicking the Request Access button and submitting an Intended Data Use statement (1-3 paragraphs in English) to be posted publicly. You will also need to review the Terms of Use and click "Accept Terms of Use". If you have questions about the data access process please post them to the [discussion forum](https://www.synapse.org/#!Synapse:syn9727752/discussion/default). <br>','If you publish work using data from this project, please: </br> <ul> <li>Acknowledge the research participants as data contributors by including the following statement: �The research team would like to thank the patients who generously shared their time and tissue for the purposes of this project.�</li> <li>Cite the study investigators on all publications or presentations resulting from the use of these data as follows: Mansouri, S., Suppiah, S., Mamatjan, Y. et al. Epigenomic, genomic, and transcriptomic landscape of schwannomatosis. Acta Neuropathol (2020). https://doi.org/10.1007/s00401-020-02230-x </li> <li>Acknowledge the NF Data Portal using the following statement: "The results published here are in whole or in part based on data obtained from the NF Data Portal (http://www.nf.synapse.org, RRID:SCR_021683) and made available through the NF Open Science Initiative."</li> </ul>',"syn9727752"]},{rowId:543,versionNumber:897,values:["The Role of MEK-inhibitors for the Prevention and Treatment of Optic Pathway Gliomas in an Nf1 Deficient Mouse Model","syn4939914","Optic pathway gliomas (OPGs) develop in 15-20% of children with NF1 and are associated with vision loss. My preliminary data, using an NF1-deficient mouse model, indicates that there are two critical steps that need to take place in order for optic pathway gliomas to form. The first step occurs early during optic nerve development, occurring before 21 days of life. The second step involves immune cell infiltration, and thus swelling of the optic nerve and loss of retinal ganglion cells (which results in vision loss). This second step occurs around 60 days of life, and continues to progress as the mouse ages. The goal of this project is to understand this process, and develop a therapeutic plan that can be used to prevent OPG formation.",null,null,"syn4939914"]},{rowId:544,versionNumber:897,values:["Regulation of GCPII for the Diagnosis and Treatment of Neurofibromas","syn4939904","Preliminary data suggest that GCPII may be useful as diagnostic marker in clinical pathology to differentiate histologically similar but clinically distinct malignant vs. benign NF1 tumors. This is of great clinical need as there is no consistent method to differentiate neurofibromas and MPNSTs pathologically. GCPII is selectively expressed on non-myelinating Schwann cells in the PNS (Berger, Luthi-Carter et al. 1999), which is thought to be the cell of origin for neurofibroma tumor development. Recent (unpublished data) demonstrated that small molecule GCPII inhibitors could enhance Schwann cell-mediated myelination both in vitro and in vivo, possibly by inducing non-myelinating Schwann cells to differentiate into a myelinating phenotype. By extension, it was considered conceivable that GCPII inhibition could limit the non-myelinating Schwann cell proliferation that occurs in neurofibromas by inducing the Schwann cells into a differentiated phenotype. The goals of this project are to evaluate (1) the Schwann cell enzyme Glutamate Carboxypeptidase II (GCPII) as a potential diagnostic marker to distinguish between malignant and benign peripheral nerve sheath tumors in patients with neurofibromatosis type 1 (NF1), and (2) the potential therapeutic effect of inhibiting GCPII enzymatic activity on the growth of neurofibroma tumors in animal models.","N/A","N/A","syn4939904"]},{rowId:545,versionNumber:897,values:["Transition to confirmatory and  secondary screening of plexiform  neurofibroma models in 96-well format","syn4939916","In our primary NTAP project we developed and optimized 3D models of NF1 plexiform neurofibromas (PNs) with the goal of providing efficient and effective systems for secondary screening of compounds that would be identified as promising new approaches for these tumors. Another primary NTAP project was performed at NCATS to screen for potential therapeutics using the CellTiter-Glo assay of proliferation as the read-out in a 1536-well 2D growth format. The goals of this project are to prove that cells grown in 3D matrices, as compared to those growing in 2D (on plastic) will exhibit drug sensitivity that is a better clinical predictor.","Accessing the data from this study requires a Synapse account.","The data from this study are still under embargo, therefore, if you have been granted access by the data contributor, you must work with them to determine how to acknowledge your collaboration in any manuscripts that arise. In addition, please acknowledge the NF Data Portal like so: “The results published here are in whole or in part based on data obtained from the NF Data Portal (http://www.nf.synapse.org, RRID:SCR_021683) and made available through the NF Open Science Initiative.”","syn4939916"]},{rowId:546,versionNumber:897,values:["Search of the molecular mechanisms sustaining plexiform neurofibroma formation in a novel three-dimensional in vitro model","syn8016577","Mechanical cues, including tissue geometry and intracellular forces, have been shown to regulate the neoplastic transformation process up to malignancy. The hypothesis of this project is that the neoplastic phenotype of Schwann cells (SCs) is fostered by the amplification of integrated signaling pathways triggered not only by loss of neurofibromin and by the ensuing Ras hyperactivation, but also by deregulation in the composition and physical properties of the extracellular matrix (ECM). The goal of this project is to determine the role of the Extracellular Matrix (ECM) in the neoplastic transformation of Schwann cells (SCs).",null,"The data from this study are still under embargo, therefore, if you have been granted access by the data contributor, you must work with them to determine how to acknowledge your collaboration in any manuscripts that arise. In addition, please acknowledge the NF Data Portal like so: “The results published here are in whole or in part based on data obtained from the NF Data Portal (http://www.nf.synapse.org, RRID:SCR_021683) and made available through the NF Open Science Initiative.”","syn8016577"]},{rowId:547,versionNumber:897,values:["Use of the mouse model of cutaneous neurofibromas for identification of cells of origin and drug screening studies","syn11374337","Despite major progress in the understanding of the mechanisms driving development of NFs, the cells of origin and scenario governing initiation of cNFs, remains unknown. We have recently shown that the neural crest derived boundary cap (BC) cells, transiently present at the interface between central and peripheral nervous system in a mouse embryo, migrate sequentially along nerve roots and along peripheral nerves under the skin, two locations where NFs develop with a higher incidence in NF1 patients. Furthermore, in the adult skin, BC derivatives correspond to various types of Schwann cells and dermal stem cells with gliogenic potential, two cell populations considered as the primary pathogenic target of NF1. Based on these observations, we hypothesized that BCs could be directly implicated in the development NFs. To explore such possibility, we have generated mouse line carrying NF1 loss combined with fluorescent tracer in BC cells and their derivatives. All mutant mice develop both paraspinal plexiform and cutaneous NFs whose location and cellular composition faithfully recapitulates NFs present in human NF1 patients.The aim of this study is to take advantage of this unique model and our extensive expertise of NF1 pathology and of mechanisms governing the development of Schwann cells to tackle three important issues concerning the initiation and progression, cellular origin and novel treatment of cNFs.","The data from this study is currently under embargo. Please contact the principal investigator for access to the data.","The data from this study are still under embargo, therefore, if you have been granted access by the data contributor, you must work with them to determine how to acknowledge your collaboration in any manuscripts that arise. In addition, please acknowledge the NF Data Portal like so: “The results published here are in whole or in part based on data obtained from the NF Data Portal (http://www.nf.synapse.org, RRID:SCR_021683) and made available through the NF Open Science Initiative.”","syn11374337"]},{rowId:548,versionNumber:897,values:["Prevention trial for tumor formation in mice modeling NF1 microdeletion patients","syn4939908","The presentation of NF1 features however is not uniform, even within patients of the same family. Genotype-phenotype correlations have thus far been unsuccessful at predicting which patients will develop what features. The only exception is patients carrying an NF1 microdeletion. These patients have a uniform germline deletion of 1.4Mb containing 14 genes, including NF1 and have a more severe phenotype. The vast majority of microdeletion patients present themselves with >1000 cutaneous and subcutaneous neurofibromas. They are at increased risk for the development of plexiform neurofibromas (76% vs 43-50%) and have a 4-fold increase in lifetime risk for MPNSTs (up to 50%); tumors also arise at a younger age. As a consequence NF1 microdeletion patients are followed up more rigorously and any complaint of pain or growing tumors is seldom ignored. NF1 microdeletion patients would benefit tremendously from the development of therapies that would prevent the onset MPNSTs and by inference of plexiform neurofibromas. In this patient group the benefits would outweigh the risk of conducting a prevention trial. The goals of this project are to determine if BRD4 inhibitors can prevent or delay the onset of tumors in a mouse model for MPNST development in NF1 microdeletion patients (NPS mice) and if the treatment with BRD4 inhibitors alters the recruitment of immune cells to tumor initiation sites. Specifically we ask if there is an enhanced recruitment of anti-tumor immune cells, like cytotoxic T-cells, or a reduction in immune suppressive myeloid cells.","N/A","N/A","syn4939908"]},{rowId:549,versionNumber:897,values:["Targeting the mechanisms underlying cutaneous neurofibroma formation in NF1: a clinical translational approach","syn4939910","Cutaneous neurofibromas (CNFs) can be distinguished from other cutaneous manifestations of NF1 based on their size (0.5-3cm), sessile or polypoid morphology, and typical lack of pigment. CNFs account for substantial morbidity and disfigurement in NF1. Unlike plexiform neurofibromas, which are congenital lesions, CNFs manifest during adolescence suggesting that their pathogenesis is influenced by environmental or hormonal factors. CNFs are comprised of a confederacy of cell types with distinct embryonic lineages, including Schwann-like cells, perineural cells (neural crest or neuro-ectoderm), fibroblasts (mesoderm), and mast cells (bone marrow-derived). The goal of this project is to develop new therapy strategies for peripheral nerve sheath tumors associated with Neurofibromatosis Type 1: Cutaneous Neurofibromas (CNFs) and Malignant Peripheral Nerve Sheath Tumors (MPNSTs).",'Accessing data from this project requires agreement to Data Access Terms of Use. <br><br> To access data from this project, please follow these steps: <br><br> **1**: Register for a Synapse account [here](https://www.synapse.org/#!RegisterAccount:0) (if you do not already have one). <br> **2**: Navigate to the <a href="https://www.synapse.org/#!Synapse:syn11642240" target="_blank">data files page</a>. <br> **3**: Click on "Request Access" next to the "lock" symbol. <br> **4**: Follow the instructions on that page. You will be prompted be prompted to review the Terms of Use and click "Accept Terms of Use". If you have questions about the data access process please email nf-osi@sagebionetworks.org. <br>','Please cite the study investigators on all publications or presentations resulting from the use of these data as follows: Grit, J.L., Johnson, B.K., Dischinger, P.S. et al. Distinctive epigenomic alterations in NF1-deficient cutaneous and plexiform neurofibromas drive differential MKK/p38 signaling. Epigenetics & Chromatin 14, 7 (2021). https://doi.org/10.1186/s13072-020-00380-6." . In addition, please acknowledge the NF Data Portal like so: “The results published here are in whole or in part based on data obtained from the NF Data Portal (http://www.nf.synapse.org, RRID:SCR_021683) and made available through the NF Open Science Initiative.”',"syn4939910"]},{rowId:550,versionNumber:897,values:["Testing the Pharmacologic role of Molecular Target inhibition on Plexiform Neurofibromas in a Genetically Engineered Mouse Model that closely Phenocopies Human Neurofibromatosis Type 1","syn8012599","In a recent gene therapy study performed in the laboratory of Dr. Xandra Breakefield, the efficacy and safety of recombinant adeno-associated viral (AAV) virus mediated delivery of Caspase-1 was tested in a schwannoma mouse model. These studies identified that direct injection of the AAV product into the sciatic schwannoma lead to the regression of well-established tumors and importantly, prevented new tumors from developing (Prabhaka et al., 2013, Human Gene Therapy). Utilizing a similar concept and in collaboration with Dr. Breakefield, we will test the efficacy of AAV serotypes and its associated control serotype in preventing (as well as reducing) existing tumor volume and number in our Nf1 plexiform neurofibroma mouse model.  Angiogenesis plays a major role in tumor development, growth, and invasion of nearby cells, wherein endoglin is expressed on tumor vessels and is essential for angiogenesis. We hypothesize that inhibition of the endoglin pathway as well as the VEGF pathway in a combinatory therapeutic setting using our Nf1 tumor bearing mouse model will provide signification tumor reduction of plexiform neurofibromas.  Neurofibromas are complex tumors composed of Schwann cells, mast cells, fibroblasts and perineurial cells embedded in collagen that provide a lattice for tumor invasion. Fibroblasts are a major cellular constituent in neurofibromas and are a source of collagen that constitutes ~50% of the dry weight of the tumor. Work performed in our lab has shown that two of the prevalent heterozygous cells found in neurofibromas, mast cells and fibroblasts interact directly to contribute to the tumorigenic phenotype. Nf1+/- mast cells secrete elevated concentrations of the profibrotic transforming growth factor-beta (TGF-beta). We will conduct studies exploring a TGF-beta inhibitor, as we foresee this as a potential drug target that could be accelerated to in human testing if successful in our preclinical work.  Goals of Project: To evaluate the therapeutic efficacy of various AAV serotypes, a TGF- beta inhibitor, a VEGF inhibitor and a VEGFR inhibitor against existing plexiform neurofibromas utilizing a novel genetically engineered mouse model of plexiform neurofibroma.","The data from this study is currently under embargo. Please contact the principal investigator for access to the data.","The data from this study are still under embargo, therefore, if you have been granted access by the data contributor, you must work with them to determine how to acknowledge your collaboration in any manuscripts that arise. In addition, please acknowledge the NF Data Portal like so: “The results published here are in whole or in part based on data obtained from the NF Data Portal (http://www.nf.synapse.org, RRID:SCR_021683) and made available through the NF Open Science Initiative.”","syn8012599"]},{rowId:551,versionNumber:897,values:["Drug-Target Explorer","syn11672851","Phenotypic high-throughput screens are often utilized in modern oncology drug discovery pipelines. Such screens are conducted with an array of molecules (panels of oncology drugs, drugs for other indications, and compounds with possibly unknown bioactivity), ultimately measuring a biological change within a disease model. While these screens can yield valuable hits, they also present the challenge of identifying the target(s) that mediate the effect seen in the screen for several reasons. For example, many drugs have a polypharmacologic target profile making it hard to identify which targets are biologically relevant for a given drug. Furthermore, some libraries contain molecules for which drug-target studies have never been conducted. To address these challenges, we developed an application that enables exploration of the chemical-biological interaction space. Drug-target interaction data from ChEMBL (CC BY-SA 3.0), ChemicalProbes.org (CC BY-SA 3.0), DGIdb (open), DrugBank (CC BY-NC 4.0, used with approval), and Klaeger, et al, 2017 was collected for over 280,000 molecules. In addition, when quantitative data were obtained, pChEMBL values were calculated. Each molecule was annotated with a name and chemical structure, and every target was annotated with protein & gene identifiers. An interactive web interface was developed using the R Shiny platform and cheminformatics R packages.","Accessing the data from this study requires a Synapse account.",'If you use data from this study in a publication or talk, please cite the following publication: "Allaway RJ, La Rosa S, Guinney J, Gosline SJC. Probing the chemical-biological relationship space with the Drug Target Explorer. J Cheminform. 2018;10(1):41. Published 2018 Aug 20. doi:10.1186/s13321-018-0297-4." In addition, please acknowledge the NF Data Portal using the following statement: "The results published here are in whole or in part based on data obtained from the NF Data Portal (http://www.nf.synapse.org, RRID:SCR_021683) and made available through the NF Open Science Initiative."',"syn11672851"]},{rowId:552,versionNumber:897,values:["NF Cell Line Compound Screens","syn11817821","This project created the first college course in drug screening, and specifically screened for drugs for NF1 and NF2 related tumor types. Students screened drugs, both known and novel, against NF tumor cell models. The known drugs will serve as a starting point for comparison with other screening efforts. In total, 130 drugs were screened against 11 different NF-related cell lines.","Accessing the data from this study requires a Synapse account.",'If you use data from this study in a publication or talk, please cite the following publication: "Guo J, Grovola MR, Xie H, et al. Comprehensive pharmacological profiling of neurofibromatosis cell lines. Am J Cancer Res. 2017;7(4):923��_934. Published 2017 Apr 1." In addition, please acknowledge the NF Data Portal using the following statement: "The results published here are in whole or in part based on data obtained from the NF Data Portal (http://www.nf.synapse.org, RRID:SCR_021683) and made available through the NF Open Science Initiative."',"syn11817821"]},{rowId:553,versionNumber:897,values:["The Development and Validation of PRO Measures to Assess Pain in Individuals with NF1 and pNF","syn4939912","Clinical trials for the treatment of PNs in children and adults with NF1 are underway. The FDA recommends including endpoints in NF1 clinical trials that assess changes in symptoms, such as pain, in addition to tumor reduction. Currently, no valid PRO measures exist that assess PN-related pain and its effect on daily functioning. The first phase of this limited-site study is to modify existing PRO pain tools using qualitative methods to ensure understanding of the concepts being assessed and to examine how the current tools work in the NF1/PN population. The second phase is to examine the psychometric properties of the modified PROs in individuals with NF1 and PNs. The goal of this project is to produce the most reliable, valid, and feasible measures of pain intensity and pain interference, as end-points in clinical trials or intervention studies.",null,"The data from this study are still under embargo, therefore, if you have been granted access by the data contributor, you must work with them to determine how to acknowledge your collaboration in any manuscripts that arise. In addition, please acknowledge the NF Data Portal like so: “The results published here are in whole or in part based on data obtained from the NF Data Portal (http://www.nf.synapse.org, RRID:SCR_021683) and made available through the NF Open Science Initiative.”","syn4939912"]}]}},lastUpdatedOn:"2024-08-17T17:19:44.287Z"},c={concreteType:"org.sagebionetworks.repo.model.table.QueryBundleRequest",entityId:"12345",partMask:g|y|b|w|v,query:{sql:"SELECT * FROM syn16787123",limit:f,offset:0,selectedFacets:[{columnName:"projectStatus",concreteType:"org.sagebionetworks.repo.model.table.FacetColumnValuesRequest",facetValues:["Active","Completed"]},{columnName:"tumorType",concreteType:"org.sagebionetworks.repo.model.table.FacetColumnValuesRequest",facetValues:[p,"Cutaneous Neurofibroma","JMML","Low Grade Glioma","MPNST","Plexiform Neurofibroma","Plexiform Neurofibroma | MPNST","Plexiform Neurofibroma | MPNST | Cutaneous Neurofibroma","Schwannoma","Schwannoma | Meningioma","SMN"]}]}},Ci={title:"Explore/SynapseTable",component:S,parameters:{stack:"mock"},decorators:[a=>t.jsx(F,{initQueryRequest:c,children:t.jsx(T,{visibleColumnCount:2,children:t.jsx(a,{})})})]},e={loaders:[()=>{u(c.query,n)}],parameters:{msw:{handlers:[N.rest.get(i+"/repo/v1/entity/syn16787123",(a,d,o)=>d(o.status(200),o.json(h))),...m(i)]}},args:{rowSet:n.queryResult.queryResults}};var r,s,l;e.parameters={...e.parameters,docs:{...(r=e.parameters)==null?void 0:r.docs,source:{originalSource:`{
  loaders: [() => {
    registerTableQueryResult(request.query, queryResultBundleJson);
  }],
  parameters: {
    msw: {
      handlers: [rest.get(MOCK_REPO_ORIGIN + '/repo/v1/entity/syn16787123', (req, res, ctx) => {
        return res(ctx.status(200), ctx.json(mockTableEntity));
      }), ...getHandlersForTableQuery(MOCK_REPO_ORIGIN)]
    }
  },
  args: {
    rowSet: queryResultBundleJson.queryResult.queryResults
  }
}`,...(l=(s=e.parameters)==null?void 0:s.docs)==null?void 0:l.source}}};const Ai=["SynapseTableDemo"];export{e as SynapseTableDemo,Ai as __namedExportsOrder,Ci as default};
